### PR TITLE
Fix string input values

### PR DESCRIPTION
**Pull request by osterman**
## What
- Properly quote all strings in inputs

## Why
- Only string values are permitted

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -17,35 +17,35 @@ inputs:
...
     description: 'Viewport width in pixels'
-    required: true
+    required: 'true'
   viewportHeight:
     description: 'Viewport height in pixels'
-    required: true
+    required: 'true'
   imageQuality:
     description: 'Quality of the output image (1-100, applicable for JPEG)'
     required: false
```
Comment (aknysh) [inline action.yml:35]: why some booleans are strings, but some not?
```
imageQuality:
    description: 'Quality of the output image (1-100, applicable for JPEG)'
    required: false
```

Comment (osterman) [reply action.yml:35]: ah, rebase. Will fix.

Comment (osterman) [inline action.yml:35]:
```suggestion
    required: 'false'
```

File: action.yml [hunk 2]
```diff
@@ -17,35 +17,35 @@ inputs:
...
-    default: png
+    default: 'png'
   omitBackground:
     description: 'Omit the browser default background. Enable to support transparency.'
-    default: true
+    default: 'true'
   viewportWidth:
     description: 'Viewport width in pixels'
-    required: true
+    required: 'true'
```
Comment (osterman) [inline action.yml]:
```suggestion
    required: true
```

File: action.yml [hunk 3]
```diff
@@ -17,35 +17,35 @@ inputs:
...
   viewportHeight:
     description: 'Viewport height in pixels'
-    required: true
+    required: 'true'
```
Comment (osterman) [inline action.yml]:
```suggestion
    required: true
```

File: action.yml [hunk 4]
```diff
@@ -17,35 +17,35 @@ inputs:
...
   imageQuality:
     description: 'Quality of the output image (1-100, applicable for JPEG)'
-    required: false
+    required: 'false'
```
Comment (osterman) [inline action.yml]:
```suggestion
    required: false
```

**Review by aknysh — APPROVED** [review]

**Pull request closed without merging**